demo/index.html

```diff
@@ -138,23 +138,21 @@ <h2 slot="header">Rich</h2>
             <style>
               [part='tooltip-title'],
               [part='tooltip-description'] {
-                text-align: left;
-                padding: 5px;
-                width: 100%;
+                padding: 0 8px;

```

Comment:
Why wouldn't padding use Lumo sizing units here?

Reply:
It's changed. But it's only used in the demo, that's probably the main reason :)